back-end/accounts/serializers.py

```
@@ -0,0 +1,63 @@
+            bio = validated_data.get('bio', ''),
+        )
+        return user
+    
+# 마이 페이지 사용자 프로필 serializer
+class ProfileSerializer(serializers.ModelSerializer):
+    # 비밀번호: PATCH 요청 시에만(설정할 때만) 비밀번호를 확인할 수 있도록 - GET 요청 시 응답에 포함 X
+    current_password = serializers.CharField(write_only=True, required=True)
+
+    class Meta:
+        model = User
+        fields = ['name', 'profile_image', 'bio', 'phone', 'current_password']
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)